accord-core/src/test/java/accord/utils/Property.java

```diff
@@ -708,6 +708,8 @@ public interface Setup<State, SystemUnderTest>
         private final Map<Setup<State, SystemUnderTest>, Integer> knownWeights = new LinkedHashMap<>();
         @Nullable
         private Set<Setup<State, SystemUnderTest>> unknownWeights = null;
+        @Nullable
+        private Map<Predicate<State>, List<Setup<State, SystemUnderTest>>> conditionalCommands = null;
```

Comment:
some of the stateful tests were flakey due to too many selections that were told to be `Ignored`... so avoid that by only including them *when* their predicate matches... this optimization was known at the time of writing this class, but going with a simpler one was selected as things were stable...